tests/whitebox/_misc.py

```diff
@@ -35,8 +35,8 @@
 try:
     _STRATISD = os.environ["STRATISD"]
 except KeyError:
-    message = "STRATISD environment variable must be set to absolute path of stratisd executable"
-    sys.exit(message)
+    _MESSAGE = "STRATISD environment variable must be set to absolute path of stratisd executable"
```

Comment:
Here, please drop the unnecessary local ```_MESSAGE```` just use the literal string as an argument to ```sys.exit_```